gaussian_example/transform_base-1.0.yaml

```diff
@@ -0,0 +1,135 @@
+%YAML 1.1
+---
+$schema: http://stsci.edu/schemas/yaml-schema/draft-01
+id: asdf://asdf-format.org/extensions/transform/schemas/transform_base-1.0
+description: >
+  Common validations for transforms.
+
+definitions:
+  parameter_value:
+    description: >
+      Parameter value.  May be a single number, or in the case of a
+      transform set, a 1D ndarray of numbers.
+    anyOf:
+      - type: number
+      - tag: asdf://asdf-format.org/extensions/core/tags/ndarray-*
+        ndim: 1
+
+  parameter:
+    description: >
+      Object that encapsulates properties of a transform parameter.
+    properties:
+      value:
+        description: >
+          Parameter value.
+        $ref: '#/definitions/parameter_value'
+      unit:
+        description: >
+          Parameter unit.
+        default: >
+          Parameter has no unit.
+        tag: asdf://asdf-format.org/extensions/core/tags/unit-*
+      bounds:
+        description: >
+          Inclusive upper and lower bounds of the parameter value.
+          Used during fitting.
+        default: >
+          Parameter value is unbounded.
+        $ref: '#/definitions/bounds'
+      fixed:
+        description: >
+          If true, this parameter's value should be fixed when fitting.
+        default: >
+          Parameter is not fixed.
+        type: boolean
+    required: [value]
+    additionalProperties: false
+
+  bounds:
+    description: >
+      Inclusive upper and lower bounds for a parameter or
+      input value
+    type: array
+    items:
+      type: number
+    minItems: 2
+    maxItems: 2
+
+  input:
+    description: >
+      Object that encapsulates properties of a transform input.
+    properties:
+      name:
+        description: >
+          Input name.
+        type: string
+      unit:
+        description: >
+          Input unit.
+        default: >
+          Input has no unit.
+        tag: asdf://asdf-format.org/extensions/core/tags/unit-*
+      bounds:
+        description: >
+          Inclusive upper and lower bounds of the input value.
+        default: >
+          Input value is unbounded.
+        $ref: '#/definitions/bounds'
+    required: [name]
+    additionalProperties: false
+
+  output:
+    description: >
+      Object that encapsulates properties of a transform output.
+    properties:
+      name:
+        description: >
+          Output name.
+      unit:
+        description: >
+          Output unit.
+        default: >
+          Output has no unit.
+        tag: asdf://asdf-format.org/extensions/core/tags/unit-*
```

Comment:
I can't think of a case where it is necessary to specify units but I may be missing some. 
I'd be fine with having `inputs` and `outputs` be optional arrays of string values.
What would be missed if so>

Reply:
I thought that it was possible in astropy to specify different return units on the model (separate from the parameter units) but looking back I think I was confused?